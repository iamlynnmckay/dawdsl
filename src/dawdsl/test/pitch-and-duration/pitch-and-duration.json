{
  "program": {
    "transport": {
      "lpb": 12
    },
    "instruments": [
      {
        "instrument_filename": "/usr/local/share/renoise-3.4.2/Library/Instruments/Keys/Space Piano.xrni"
      }
    ],
    "patterns": [
      {
        "duration": 16
      }
    ],

    "events": [
      {
        "pitch": "ionian.C#.4.perfect_fifth",
        "duration": "1/2"
      },
      {
        "pitch": ["ionian.C#.4.first", "ionian.C#.4.second"],
        "duration": ["1/4", "1/4"]
      }
    ]
  },
  "test": {
    "expected": "./test/pitch-and-duration/pitch-and-duration.expected.json",
    "actual": "./test/pitch-and-duration/pitch-and-duration.actual.json"
  }
}
