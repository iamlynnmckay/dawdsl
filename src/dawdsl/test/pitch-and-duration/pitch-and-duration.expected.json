{
  "program": [
    "renoise.song().transport.lpb = 8",
    "renoise.song().tracks[1].visible_note_columns = 1",
    "renoise.song().tracks[1].visible_effect_columns = 0",
    "renoise.song().instruments[1]:clear()",
    "renoise.song().patterns[1].number_of_lines = 64"
  ]
}
